local-volume/README.md

```diff
@@ -83,6 +83,13 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 3. Continue with [Running the external static provisioner](#running-the-external-static-provisioner)
    below.
 
+
+#### GCE E2E Testing
+``` console
+KUBE_FEATURE_GATES="PersistentLocalVolumes=true" NODE_LOCAL_SSDS=1 go run hack/e2e.go -- -v --up
```

Comment:
The e2e should be able to run against any env, not just gce, so I would remove this line since it is already mentioned above.